Firestore/core/test/firebase/firestore/util/status_test.cc

```diff
@@ -113,6 +113,18 @@ TEST(Status, FromErrno) {
           "Already exists: Cannot write file \\(errno .*: File exists\\)"));
 }
 
+TEST(Status, Annotate) {
+  Status a(FirestoreErrorCode::NotFound, "fork not found");
+  Status b = a.Annotate("could not eat dinner");
+  ASSERT_EQ(FirestoreErrorCode::NotFound, b.code());
+  ASSERT_EQ("fork not found; could not eat dinner", b.error_message());
```

Comment:
Since you don't guarantee the separator, you might instead do:
```
ASSERT_NE(b.error_message().find("fork not found") , string::npos);
ASSERT_NE(b.error_message().find("could not eat dinner"), string::npos);
```

(Maybe gtest has some string/container "contains" helper to make this better.)

Reply:
There's some string matchers in gmock. I'll use those in the follow-up PR. This has been backed out.